src/stores/SearchStore.ts

```diff
@@ -129,6 +129,12 @@ export class SearchStore {
 				}
 			},
 		);
+
+		reaction(
+			() => this.searchForm.parentEvent,
+			parentEvent =>
+				parentEvent && !isEventId(parentEvent) && this.loadEventAutocompleteList(parentEvent),
```

Comment:
clear autocompleteList if there is no parent event.
Event was copy/paste, but since we now accept event name i think we should debounce it